src/PointwiseFunctions/Xcts/AdmMass.hpp

```diff
@@ -52,4 +52,74 @@ tnsr::I<DataVector, 3> adm_mass_surface_integrand(
     const tnsr::i<DataVector, 3>& conformal_christoffel_contracted);
 /// @}
 
+/// @{
+/*!
+ * \brief Volume integrand for the ADM mass calculation.
+ *
+ * We cast the ADM mass as an infinite volume integral by applying Gauss' law on
+ * the surface integral defined in `adm_mass_surface_integrand`:
+ *
+ * \begin{equation}
+ *   M_\text{ADM} = \frac{1}{16\pi}
+ *                  \int_{V_\infty} \Big(
+ *                    \bar\gamma^{jk} \partial_i \bar\Gamma^i_{jk}
+ *                    + \bar\Gamma^i_{jk} \partial_i \bar\gamma^{jk}
+ *                    - \bar\gamma^{ij} \partial_i \bar\Gamma_j
+ *                    - \bar\Gamma_j \partial_i \bar\gamma^{ij}
```

Comment:
If I understand it right, these extra terms would only show up if we take the covariant derivative $\bar D_i$ when applying Gauss' law instead of a partial derivative $\partial_i$ (this is what I'm currently doing). I remember we briefly talked about these two options (referencing Saul's recent DG paper), and we decided that $\partial_i$ was the way to go.

That said, after redoing the calculations, I found an extra term involving the conformal factor first derivative:
$\partial_i ( - 8 \bar\gamma^{ij} \partial_j \psi ) = - 8 \bar\gamma^{ij} \partial_i \partial_j \psi - 8 \partial_i \bar\gamma^{ij} \partial_j \psi = - 8 \bar D^2 \psi - 8 \partial_i \bar\gamma^{ij} \partial_j \psi $.
Again, this is only if we use $\partial_i$ in Gauss' Law. Looking at Baumgarte-Shapiro's equation (3.146), they seem to use $\bar D_i$ in Gauss' Law.

I'm not sure what approach we should take here, but both of them require updates to these integrands.

Reply:
You need the covariant Laplacian of $\psi$ to replace it with the Hamiltonian constraint, so I would go with the covariant derivatives and the Christoffel terms I wrote above. As you say, either way you have to account for the covariant derivatives with Christoffel terms.

Reply:
Ok, that makes sense. This raises another point. If we're using $\bar D_i$ when applying Gauss' law, then can we still assume a flat space in this transformation?

Reply:
For Gauss' law you transform $\oint n_i \sqrt{\gamma} F^i = \int \partial_i (\sqrt{\gamma} F^i) = \int \sqrt{\gamma} \nabla_i F^i$ (see Eqs. (7-10) in https://arxiv.org/abs/physics/9802027v1). So you get the covariant derivative from moving the metric determinant through the partial derivative.
So, you integrate with the curved area/volume element, get a covariant derivative from Gauss' law, and expand it into Christoffel terms where needed.

Reply:
This is helpful, thanks! The test that I had in `Test_AdmMass` was using flat area/volume elements and started failing after these changes. Once I updated this test to use curved area/volume elements, I started getting the correct values again. So, things seem consistent.

Reply:
Ok great! Both approaches work, we just have to be consistent. Please note in the docs for the integrand function if it needs to be integrated with the flat or conformal area/volume element.